.github/workflows/pages.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           cache-version: 0 # Increment this number if you need to re-download cached gems
           working-directory: '${{ github.workspace }}/docs'
       - name: Setup Pages
+        if: ${{ github.event_name == 'push' && github.repository == 'GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration' && github.ref_name == github.event.repository.default_branch}}
```

Comment:
We need to add an `workflow_dispach` to the allowed events, otherwise we cannot trigger this manually.

`(github.event_name == 'push' || github.event_name == 'workflow_dispach') && ....`

Reply:
Do this for the other `if`s as well.

Reply:
Ok. Let's try this: update the other `if`s as I described, and remove this one.